modules/programs/neomutt.nix

```diff
@@ -234,7 +310,9 @@ let
       ${account.neomutt.extraConfig}
 
       ${signature}
-    '' + optionalString account.notmuch.enable (notmuchSection account);
+    ''
+    + optionalString (account.notmuch.enable && account.notmuch.neomutt.enable)
+    (notmuchSection account);
```

Comment:
I think the check here is invalid. The "notmuch" section should only be generated if `notmuch.enable == true`.

Reply:
The `&&` side enable you to selectively disable notmuch on a specific account, I have many accounts where I prefer dumb mailboxes rather than a full-fledged Notmuch section.